src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -798,31 +893,199 @@ let replaceOutputColumn (annotationTableName:string) (existingOutputColumn: Buil
         }
     )
 
+/// <summary>
+/// Update an existing inputcolumn of an annotation table
+/// </summary>
+/// <param name="excelTable"></param>
+/// <param name="arcTable"></param>
+/// <param name="newBB"></param>
+let updateInputColumn (excelTable:Table) (arcTable:ArcTable) (newBB:CompositeColumn) =
+
+    let possibleInputColumn = arcTable.TryGetInputColumn()
+    let inputColumnName =
```

Comment:
Do not default to column name to "" if it is empty, just work with options. No reason to run the code below if we get a None here.

```fsharp
Option.map (fun c -> c.Header.ToString())
|> Option.map (fun cName -> excelTable.columns.item.Find (fun s -> ...)
```

Reply:
Same for output

Reply:
fixed